Pod/Classes/Cards/Default Card Types/Discover.swift

```diff
@@ -14,7 +14,7 @@ public struct Discover: CardType {
     
     public let CVCLength = 3
     
-    public let identifyingDigits = Set(644...649).union( Set(622126...622925) ).union( Set([6011]) )
+    public let identifyingDigits = Set(644...649).union( Set(622126...622925) ).union( Set([6011]) ).union(Set([65]))
```

Comment:
According to https://en.wikipedia.org/wiki/Bank_card_number
<img width="906" alt="screen shot 2016-03-31 at 10 34 47 am" src="https://cloud.githubusercontent.com/assets/5505758/14184902/4efa80da-f72c-11e5-8555-f8ffa042888b.png">
Discover has one more set of 65